Vagrantfile

```diff
@@ -245,6 +246,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       os.ssh_username = user = voc['ssh_user']|| creds['OSSshUser']  || "root"           # login for the VM instance
       os.server_name  = ENV['OS_HOSTNAME']    || vagrant_openshift_config['instance_name'] # name for the instance created
       full_provision(override.vm, user)
+      floating_ip_pool = creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']
```

Comment:
For floating_ip, do you really want it to be a sym in one case (:auto) and not in the other?  Meaning is this wrong:

os.floating_ip  = (creds['OSFloatingIP'] || ENV['OS_FLOATING_IP']).to_sym

2 lines is ok for floating_id_pool.